src/main/java/com/flightstats/hub/webhook/WebhookResource.java

```diff
@@ -168,4 +170,29 @@ static Response deleter(@PathParam("name") String name) {
         webhookService.delete(name);
         return Response.status(Response.Status.ACCEPTED).build();
     }
+
+    @Path("/{name}/updateCursor")
+    @PUT
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    public Response updateCursor(@PathParam("name") String name, String body) {
+        return cursorUpdater(name, body, uriInfo);
+    }
+
+    static Response cursorUpdater(@PathParam("name") String name, String body, UriInfo uriInfo) {
+        logger.info("update cursor webhook {} {}", name, body);
+        Webhook webhook = Webhook.fromJson(body, webhookService.get(name)).withName(name);
+        ObjectMapper mapper = new ObjectMapper();
+        try {
+            JsonNode root = mapper.readTree(body);
+            if (root.has("item")) {
```

Comment:
Instead of passing in a JSON object with the URL...

    {"item": "http://some/item/url"}

.. how about just a plain-text body with the URL?

    http://some/item/url